tools/extract_changelog.py

```diff
@@ -33,6 +33,8 @@ def main():
         i = i + 1
 
     changelog = [entry for entry in changelog if entry]
+    if changelog[-1] == "More details about these changes can be found on our GitHub repo.":
```

Comment:
In fact my intention with `changelog = [entry for entry in changelog if entry]` was to remove any leading or trailing new lines in the output. But this also removes any empty lines in between, which can participate to the final text formating in markdown syntax.

So to make your fixe more generic, you can just remove line 35, and it will be sufficient to fix the issue. If you want to go a step further, you can maybe remove the first element in `changelog` if it is an empty string, to remove the first new line in the output.

Reply:
Thank you for confirming it. I updated the pull request.

- Remove line 35 to keep empty lines.
- Update line 27 to remove the first newline.